src/containers/search/filterSelector/excludeLink.js

```diff
@@ -0,0 +1,42 @@
+import {markAsExcluded, stringifyQuery} from '../../../lib/searchHelpers';
+import {Link} from 'react-router-dom';
+import React from 'react';
+import * as PropTypes from 'prop-types';
+
+/**
+ * @return {null}
+ */
+function ExcludeLink(props) {
+  let {queryObject, value, displayExclude, SEARCH_PATH, toggleStrikeThrough, displayName} = props;
+  let newQueryObject = markAsExcluded(queryObject, value);
+  let visibility = displayExclude ? 'visible' : 'hidden';
+  if (displayName !== 'Category') {
+    return (
+      <span
+        style={{visibility: visibility}}
+        onMouseEnter={toggleStrikeThrough}
+        onMouseLeave={toggleStrikeThrough}
+      >
+        <Link
+          to={{pathname: SEARCH_PATH, search: stringifyQuery(newQueryObject)}}
+        >
+          <span title="Exclude term">&times;</span>
```

Comment:
It's probably worth doing an explicit tooltip here, the title from the span is a little slow to show up. I also noticed in the site we borrowed the idea from that they use a block level element, which seems like it gives a little bit bigger target area to click than just the x symbol itself.

Reply:
Gotcha, that makes sense. Will do.